tests/unit/test_tag_google_analytics_gtag.py

```diff
@@ -59,6 +59,21 @@ def test_identify(self):
         r = GoogleAnalyticsGTagNode().render(Context({'user': User(username='test')}))
         assert "gtag('set', {'user_id': 'test'});" in r
 
+    def test_identity_context(self):
+        """ Test when identity is set in the context """
+        r = GoogleAnalyticsGTagNode().render(Context({
+            'google_analytics_gtag_identity': 'foo',
+            'analytical_identity': 'bar',
+            'user': User(username='test'),
+        }))
+        assert "gtag('set', {'user_id': 'foo'});" in r
+
+        r = GoogleAnalyticsGTagNode().render(Context({
+            'analytical_identity': 'bar',
+            'user': User(username='test'),
+        }))
+        assert "gtag('set', {'user_id': 'bar'});" in r
```

Comment:
Can you split this up into two separate tests? IIUC, the two tests don't build on each other.

Reply:
OK